### PR TITLE
added components to Home screen

### PR DESCRIPTION
**Pull request by NetanMangal**

### CODE REVIEW

**Comment by nagpalm7** [comment]
Resolve conflicts @NetanMangal

**Comment by nagpalm7** [comment]
Add map key as env variables search google how to do that

File: src/Pages/Home/Home.js [hunk 1]
```diff
@@ -1,8 +1,272 @@
...
+        }
+      });
+
+    setState({
+      ...state,
+      locations: locs.data,
+      loading: false
+    });
+
+    message.info(`Showing data of district ${state.districts[22 - parseInt(e.key)].district}`);
```
Comment (nagpalm7) [inline src/Pages/Home/Home.js:239]: Why 22-e.key ?

**Comment by NetanMangal** [comment]
Pull request kyu band kari hai yee wali ??

**Comment by nagpalm7** [comment]
Merge krdi hai bro

**Pull request merged**